info.smplayer.SMPlayer.yml

```diff
@@ -0,0 +1,199 @@
+app-id: info.smplayer.SMPlayer
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.15'
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+rename-desktop-file: smplayer.desktop
+command: smplayer
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=fallback-x11'
```

Comment:
you only need this permission if you're enabling wayland as well (and it's used to select wayland by default). If you're not enabling wayland I'd remove it.

Reply:
```suggestion
```